? patch
? comedi/drivers/patch
? comedi/drivers/pcmad.c
? comedi/drivers/diff_ni-E.c
? include/comedi
? include/modbuild
Index: comedi/Config.in
===================================================================
RCS file: /d/ds/cvsroot/comedi/comedi/Config.in,v
retrieving revision 1.20
diff -u -r1.20 Config.in
--- comedi/Config.in	2000/08/09 20:44:20	1.20
+++ comedi/Config.in	2000/09/03 01:54:35
@@ -87,6 +87,7 @@
 if [ "$CONFIG_PCI" = "y" ];then
 	dep_tristate 'IOtech DaqBoard/2000' CONFIG_COMEDI_DAQBOARD2000 $CONFIG_COMEDI
 fi
+dep_tristate 'Winsystems PCM-A/D' CONFIG_COMEDI_PCMAD $CONFIG_COMEDI
 dep_tristate 'Skeleton driver' CONFIG_COMEDI_SKEL $CONFIG_COMEDI
 
 if [ "$CONFIG_COMEDI_RT" = "y" ];then
Index: comedi/drivers/Makefile
===================================================================
RCS file: /d/ds/cvsroot/comedi/comedi/drivers/Makefile,v
retrieving revision 1.14
diff -u -r1.14 Makefile
--- comedi/drivers/Makefile	2000/07/24 19:43:29	1.14
+++ comedi/drivers/Makefile	2000/09/03 01:54:36
@@ -39,6 +39,8 @@
 
 obj-$(CONFIG_COMEDI_II_PCI20KC)		+= ii_pci20kc.o
 
+obj-$(CONFIG_COMEDI_PCMAD)		+= pcmad.o
+
 obj-$(CONFIG_COMEDI_MULTIQ3)		+= multiq3.o
 
 obj-$(CONFIG_COMEDI_NI_ATMIO)		+= ni_atmio.o
